### DISCUSSION TITLE
Deploying Remix app from kitchen sink example to Vercel

### DISCUSSION THREAD

**Discussion by coreylight** [vercel/turborepo#448] [Help] [answered]
When I clone the kitchen sink example, I can't seem to get it deployed to Vercel correctly.
Either I get 404, or an error from my serverless function:
`Cannot find module '@remix-run/vercel'`
I'm not sure how to allow access to this necessary node lib to run the serverless function.
What settings are needed to be able to fully deploy?
Thank you!

**Comment by lukebennett88** [comment]
I've noticed this too.
Remix have an example Remix project in their repo, and it looks like they had to use rollup to get around some Vercel specific stuff (explained in the README).
https://github.com/remix-run/remix/tree/main/examples/turborepo-vercel
@jaredpalmer maybe the kitchen-sink example should be updated to add rollup (unless there is a better way)?
I'm happy to PR a fix (if this is the accepted way to use Remix in a monorepo with Vercel).

**Reply by chuckstock** [reply]
I've tried that example from the repo and still get the 404 error on Vercel. Would be great to be able to deploy a remix app from a turborepo, but I haven't seen a working example yet.

**Comment by dohomi** [comment]
same issue for me as well. remix seems not to work in a monorepo structure

**Reply by mocon** [reply]
Same here, having the 404 issue from Nx -> Remix app generated with `@nrwl/remix`. Build works fine but then shows output with no API directory and 404s.

**Comment by anthonyshew** [comment, marked as answer]
We have updated all our examples, including the `kitchen-sink`! Going to close this as stale.

Check out the new `kitchen-sink` here: https://github.com/vercel/turbo/tree/main/examples/kitchen-sink
